config/contracts.json

```diff
@@ -21,13 +21,14 @@ module.exports = {
         "args": ["$ENSRegistry"],
         "deploy": true
       },
-      "ENSSubdomainRegistry": {
+      "UsernameRegistrar": {
         "args": [
           "$TestToken",
           "$ENSRegistry",
           "$PublicResolver",
-          "3",
-          [merkleRoot],
+          "0x5f7791d31ca0493e9ca7c9ca16695ecd9d5044768674d14d31ab5d8277518fff",
+          3,
+          [],
```

Comment:
When does it get supplied the merkleRoot?

Reply:
Yet, does not. This part is missing. One of the next things I need to do :)